src/app/router/ui/RootRoute.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import { Outlet } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+export const RootRoute = () => {
+  const { t } = useTranslation(['translation']);
+  const [count, setCount] = useState<number>(0);
+  const increment = () => setCount((prev) => prev + 4);
+
+  return (
+    <React.Fragment>
```

Comment:
just for your infomration, you can use <></> instead of <React.Fragment></React.Fragment>